sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/implementation/EventHubImpl.java

```diff
@@ -383,7 +383,7 @@ public CaptureSettings withNewStorageAccountForCapturedData(
                     StorageAccount storageAccount = (StorageAccount) indexable;
                     ensureSettings().destination().withStorageAccountResourceId(storageAccount.id());
                     return createContainerIfNotExistsAsync(storageAccount, containerName)
-                        .flatMap(aVoid -> context.voidMono());
+                        .flatMap(blobContainer -> context.voidMono());
```

Comment:
blobContainer is a indexable, don't flatmap to void

Reply:
updated.